src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,17 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Size;
+import mate.academy.spring.validation.Email;
+import mate.academy.spring.validation.Password;
+
+@Password
 public class UserRequestDto {
+    @Email
     private String email;
+    @Size(min = 6)
     private String password;
+    @Size(min = 6)
+    private String repeatPassword;
```

Comment:
All password validation including size should be inside `@Password`
```suggestion
    private String password;
    private String repeatPassword;
```
![image](https://github.com/mate-academy/jv-spring-security/assets/20326858/b9892228-abf6-4fd5-ab20-9f37e511856c)